signer-service/src/api/services/pendulum.service.js

```diff
@@ -56,7 +62,32 @@ exports.sendStatusWithPk = async () => {
   const { fundingAccountKeypair, fundingAmountRaw } = getFundingData(apiData.ss58Format, apiData.decimals);
   const { data: balance } = await apiData.api.query.system.account(fundingAccountKeypair.address);
 
-  if (Big(balance.free.toString()).gte(fundingAmountRaw)) {
+  const tokensToCheck = Object.keys(TOKEN_CONFIG);
+  let isTokensSufficient = true;
+
+  // Wait for all required token balances check.
+  await Promise.all(
+    tokensToCheck.map(async (token) => {
+      const tokenConfig = TOKEN_CONFIG[token];
```

Comment:
Nit: We could also do 
```suggestion
    Object.entries(TOKEN_CONFIG).map(async ([token, tokenConfig]) => {
```